ui/app/AppLayouts/Wallet/popups/swap/SwapInputParamsForm.qml

```diff
@@ -56,9 +56,7 @@ QtObject {
         return !!root.selectedAccountAddress &&
                 root.selectedNetworkChainId !== -1 &&
                 !!root.fromTokensKey && !!root.toTokenKey &&
-                (!!root.fromTokenAmount &&
-                 !isNaN(bigIntNumber) &&
-                 SQUtils.AmountsArithmetic.cmp(bigIntNumber, 0) === 1) &&
```

Comment:
This was `assert`ing on the `cmp` call, because:

```qml
function cmp(amount1, amount2) {
        console.assert(amount1 instanceof Big.Big)
        console.assert(amount2 instanceof Big.Big)
...
```
`0` is not a Big number object